setup.py

```diff
@@ -19,7 +19,7 @@
 import sys
 import pkg_resources
 
-__version__ = '2.1.5'
+__version__ = '2.1.6'
```

Comment:
This change should be removed. Publication automation bumps version automatically.